tests/testcases/config-variables/src/lib.rs

```diff
@@ -12,15 +12,21 @@ fn handle_request(req: Request) -> Result<Response> {
         .query()
         .expect("Should have a password query string");
     let query: std::collections::HashMap<String, String> = serde_qs::from_str(query)?;
-    let provided_password = query
+    let expected_password = query
```

Comment:
I initially found this rename very very confusing because in terms of _password_ logic, this is not "expected", this is indeed the user's attempt.  But in terms of what we are testing, I agree this is the "value we expect to get from config."

So I'd suggest decoupling the names from the "password" fiction, and call this something like "expected_config_value".

Reply:
done.